### PR TITLE
Fix in_pass_encoder:subresources_and_binding_types_combination_for_aspect:*

### PR DESCRIPTION
**Pull request by austinEng**
- Remove cases that do not specify an aspect for sampling a multiplanar
  format
- Choose 'uint' as the GPUTextureSampleType when sampling stencil
- Pass the depthStencilFormat to the render bundle encoder to
  match the GPURenderPassDescriptor

Bug: crbug.com/dawn/993

<hr>

**Author checklist for test code/plans:**

- [x] All outstanding work is tracked with "TODO" in a test/file description or `.unimplemented()` on a test.
- [x] New helpers, if any, are documented using `/** doc comments */` and can be found via `helper_index.txt`.
- [x] (Optional, sometimes not possible.) Tests pass (or partially pass without unexpected issues) in an implementation. (Add any extra details above.)

**[Reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md) for test code/plans:** (Note: feel free to pull in other reviewers at any time for any reason.)

- [ ] The test path is reasonable, the [description](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) "describes the test, succinctly, but in enough detail that a reader can read only the test plans in a file or directory and evaluate the completeness of the test coverage."
- [ ] Tests appear to cover this area completely, except for outstanding TODOs. Validation tests use control cases.
    (This is critical for coverage. Assume anything without a TODO will be forgotten about forever.)
- [ ] Existing (or new) test helpers are used where they would reduce complexity.
- [ ] TypeScript code is readable and understandable (is unobtrusive, has reasonable type-safety/verbosity/dynamicity).

### CODE REVIEW

**Comment by austinEng** [comment]
Note: this PR should be stacked after #654

**Review by kainino0x — APPROVED** [review]
Last commit LGTM

File: src/webgpu/api/validation/resource_usages/texture/in_pass_encoder.spec.ts [hunk 1]
```diff
@@ -548,6 +563,9 @@ g.test('subresources_and_binding_types_combination_for_aspect')
       .combine('binding0InBundle', [false, true])
       .combine('binding1InBundle', [false, true])
       .combine('format', kDepthStencilFormats)
+      .unless(({ format }) => {
+        return format === 'depth16unorm' || format === 'stencil8';
+      })
```
[the remark this answers is not in the document]
Comment (austinEng) [reply src/webgpu/api/validation/resource_usages/texture/in_pass_encoder.spec.ts]: whoops, that was just my own testing. removed.

File: src/webgpu/api/validation/resource_usages/texture/in_pass_encoder.spec.ts [hunk 2]
```diff
@@ -644,6 +670,8 @@ g.test('subresources_and_binding_types_combination_for_aspect')
     const encoder = t.device.createCommandEncoder();
     // Color attachment's size should match depth/stencil attachment's size. Note that if
     // type1 !== 'render-target' then there's no depthStencilAttachment to match anyway.
+    const depthStencilFormat = !compute && type1 !== 'render-target' ? undefined : format;
```
[the remark this answers is not in the document]
Comment (austinEng) [reply src/webgpu/api/validation/resource_usages/texture/in_pass_encoder.spec.ts]: done

**Pull request merged**